src/events/app/index.ts

```diff
@@ -126,42 +126,46 @@ export async function handleAppInstallOnRepoEvent(
 }
 
 /**
- * General handling for syncing repositories.
+ * General handling for syncing repositories of all owners.
  * @param repoKeys
- * @param github
- * @param log
+ * @param app
  * @param pullService
  * @param issueService
  * @param commentService
  * @param contributorService
  */
 async function handleSyncRepos(
   repoKeys: RepoKey[],
-  github: InstanceType<typeof ProbotOctokit>,
-  log: Logger,
+  app: Probot,
   pullService: IPullService,
   issueService: IIssueService,
   commentService: ICommentService,
   contributorService: IContributorService
 ) {
   for (const repoKey of repoKeys) {
+    const nonAuthGithub = await app.auth();
```

Comment:
I don't think nonAuthGithub is a good name.
By the way, it should be GitHub not Github.

Reply:
Maybe `gitHubApp`?

Reply:
What name is used for the underlying library?What name is used for the underlying library?

Reply:
![image](https://user-images.githubusercontent.com/5086433/108314920-9ffcc080-71f5-11eb-8403-456c97d3fc86.png)

`octokit`, named by probot itself.

Reply:
<img width="445" alt="image" src="https://user-images.githubusercontent.com/29879298/108321951-cf182f80-71ff-11eb-8da3-b470d1dee111.png">

Maybe it's OK. But in docs we should all use GitHub.